### PR TITLE
Hotfix remove ape3

### PR DESCRIPTION
**Pull request by gabrielfior**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "prediction-market-agent-tooling"
-version = "0.48.0"
+version = "0.47.2"
```
Comment (kongzii) [inline pyproject.toml:3]: Golden rule of versioning: It only goes up

File: pyproject.toml [hunk 2]
```diff
@@ -37,15 +37,16 @@ google-api-python-client = { version = "2.95.0", optional = true}
 subgrounds = "^1.9.1"
 loguru = "^0.7.2"
 safe-eth-py = "^6.0.0b14"
-eth-account = "^0.11.2"
+eth-account = "^0.8.0"
```
Comment (kongzii) [inline pyproject.toml:40]: In my PMAT PR I did `eth-account = ">=0.8.0,<0.12.0"` and it worked fine in PMA

File: pyproject.toml [hunk 3]
```diff
@@ -37,15 +37,16 @@ google-api-python-client = { version = "2.95.0", optional = true}
...
 prompt-toolkit = "^3.0.43"
 safe-cli = "^1.0.0"
 langfuse = "^2.27.1"
 openai = { version = "^1.0.0", optional = true}
 tavily-python = "^0.3.9"
 sqlmodel = "^0.0.21"
 psycopg2-binary = "^2.9.9"
-base58 = ">=1.0.2,<2.0"
+base58 = "^2.1.1"
```
Comment (kongzii) [inline pyproject.toml:48]: If you change the eth-account version, are the rest of the changes needed?

**Comment by gabrielfior** [comment]
Closed in favor of https://github.com/gnosis/prediction-market-agent-tooling/pull/351

**Pull request closed without merging**